README

```diff
@@ -66,11 +66,16 @@ Much, much more information is also available in the Open MPI FAQ:
 ===========================================================================
 
 The following abbreviated list of release notes applies to this code
-base as of this writing (February 2019):
+base as of this writing (May 2020):
 
 General notes
 -------------
 
+- WARNING - Open MPI v4.0.3 accidentally dropped some external constant
+  definitions from the libmpi_usempif08.so library.  This is remedied
+  in v4.0.4.  It is recommended that users of libmpi_usempif08.so skip
+  over v4.0.3, and upgrade to v4.0.4 or later.
+
```

Comment:
Per https://github.com/open-mpi/ompi/pull/7696#issuecomment-624857209, should probably also add something here at the new F08 symbols that were (effectively) added in 4.0.4, and what it means to the backwards compatibility guarantee for those symbols.

Reply:
I didn't list the constants out as the list appears to be ALL of the MPI constants, and is quite long.

Reply:
No, that's not what I meant.  I mean that you should mention that if an app uses these symbols and compiles/links against 4.0.4 (i.e., the symbols that were erroneously left out of 4.0.0-4.0.3), they will not be able to link against the Fortran libraries from 4.0.0-4.0.3.

Reply:
@gpaulsen and I chatted in Slack and came up with better wording.  He will push shortly.